### PR TITLE
Manage the ceapex PAT for OneLocBuild

### PR DESCRIPTION
**Pull request by jonfortescue**
### To double check:

* [ ] The right tests are in and and the right validation has happened.  Guidance:  https://github.com/dotnet/arcade/tree/main/Documentation/Validation

### CODE REVIEW

File: .vault-config/product-builds-engkeyvault.yaml [hunk 1]
```diff
@@ -138,6 +138,18 @@ secrets:
...
+  dn-bot-ceapex-package-r:
+    type: azure-devops-access-token
+    parameters:
+      domainAccountName: dn-bot
+      domainAccountSecret:
+          location: helixkv
+          name: dn-bot-account-redmond
+      name: dn-bot-ceapex-package-r
+      organizations: ceapex
+      scopes: packaging_read
```
Comment (garath) [inline .vault-config/product-builds-engkeyvault.yaml:151]:
```suggestion
      scopes: packaging
```

Read-only scope doesn't use the suffix.

**Review by garath — CHANGES_REQUESTED** [review]
I also ran this manually to force-rotate.

**Comment by riarenas** [comment]
Would you mind moving this PAT to https://github.com/dotnet/arcade/blob/main/.vault-config/product-builds-dnceng-pipeline-secrets.yaml? Let's not add new PATs used by automation to engKeyvault

**Comment by jonfortescue** [comment]
@riarenas I don't mind moving it at all and indeed considered that I should do so. Will be taking care of that.

**Comment by jonfortescue** [comment]
Just to clarify @riarenas, this isn't a new PAT but rather an existing one that was previously in engkeyvault. Still moving it bc it should be moved.

**Comment by jonfortescue** [comment]
In order to migrate this, I'll also need to migrate BotAccount-dotnet-bot-repo-PAT. I'll be doing that shortly.

**Comment by riarenas** [comment]
> Just to clarify @riarenas, this isn't a new PAT but rather an existing one that was previously in engkeyvault. Still moving it bc it should be moved.

Ah, that makes sense. Thanks for moving it anyways!

**Review by garath — DISMISSED** [review]

**Comment by missymessa** [comment]
@jonfortescue do you recall if this work is still needed?

**Comment by jonfortescue** [comment]
@missymessa Yes, as long as we're using OneLocBuild, this will be needed. Idk if this is the right "form" for it anymore, but the work needs to be done regardless.

**Comment by garath** [comment]
@jonfortescue sorry to bother, but what was this for? To get these PATs in secret-manager? 

Any recommendations for a testing proceedure?

Is your comment about "form" regarding the format that OneLoc wants, or the form that secret-manager wants?

**Comment by jonfortescue** [comment]
@garath no worries! It was indeed for getting this PAT in secret manager. For testing, as long as it is in the correct vaults, it shouldn't be an issue -- you'll know if it breaks if the PAT expires or if you move it incorrectly bc loc will break everywhere haha (specifically, the OneLocBuild task will begin failing). The big thing would be to try to rotate it once and make sure that works.

For "form", I'm referring to the format that secret-manager wants. Given that there's merge conflicts and that I think you've made changes to secret-manager, I think that should be reevaluated, basically.

**Comment by riarenas** [comment]
I filed https://github.com/dotnet/arcade/pull/12020 to track this work. We can use this PR as a guide for the changes but this has a lot of conflicts with the current manifests, so closing.

**Pull request closed without merging**